tests-extension/.openshift-tests-extension/openshift_payload_olmv0.json

```diff
@@ -233,11 +233,588 @@
   },
   {
     "name": "[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:68679-[OTP][Skipped:Disconnected]catalogsource with invalid name is created",
-    "originalName": "[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:68679-[Skipped:Disconnected]catalogsource with invalid name is created",
     "labels": {
       "Extended": {},
-      "NonHyperShiftHOST": {},
-      "original-name:[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:68679-[Skipped:Disconnected]catalogsource with invalid name is created": {}
+      "NonHyperShiftHOST": {}
+    },
+    "resources": {
+      "isolation": {}
+    },
+    "source": "openshift:payload:olmv0",
+    "lifecycle": "blocking",
+    "environmentSelector": {
+      "exclude": "topology==\"External\""
+    }
+  },
+  {
+    "name": "[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:83027-[OTP]-Unnecessary churn with operatorgroup clusterrole management[Serial]",
+    "labels": {
+      "Extended": {}
+    },
+    "resources": {
+      "isolation": {}
+    },
+    "source": "openshift:payload:olmv0",
+    "lifecycle": "blocking",
+    "environmentSelector": {}
+  },
+  {
+    "name": "[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:70162-[OTP]-Leverage Composable OpenShift feature to make OperatorLifecycleManager optional",
+    "labels": {
+      "Extended": {}
+    },
+    "resources": {
+      "isolation": {}
+    },
+    "source": "openshift:payload:olmv0",
+    "lifecycle": "blocking",
+    "environmentSelector": {}
+  },
+  {
+    "name": "[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:73201-[OTP]-[Skipped:Disconnected]73201-catalog pods do not recover from node failure [Disruptive][Serial]",
+    "labels": {
+      "Extended": {}
+    },
+    "resources": {
+      "isolation": {}
+    },
+    "source": "openshift:payload:olmv0",
+    "lifecycle": "blocking",
+    "environmentSelector": {}
+  },
+  {
+    "name": "[sig-operator][Jira:OLM] OLMv0 optional should PolarionID:72192-[OTP]-is not correctly refreshing operator catalogs due to IfNotPresent imagePullPolicy",
+    "labels": {
+      "Extended": {},
+      "LEVEL0": {}
```

Comment:
@jianzhangbjz LEVEL0 is not ginkgo label. it should be title label '[Level0]'

same comment for other cases.